content/gamenews

```diff
@@ -1,4 +1,4 @@
-2015-04-08 12-00-00:
-	Title: Red Alert balancing finished for the next release!
+2015-04-12 12-00-00:
+	Title: New OpenRA release out now! This is not a drill!
 	Author: Oliver Brakmann
-	Content: The Red Alert balancing adjustments have been finished and thus warrant another playtest, which will hopefully be the last before our next long-awaited release.\n\nBesides the RA balancing, which represents the bulk of the changes, some minor fixes and polishing are included. For the detailed list, please see our changelog at http://changelog.openra.net as usual.\n\nThis will likely be your last chance to give feedback and contribute to this release, so please report any problems you encounter at http://bugs.openra.net.\n\nThank you, and have fun!
+	Content: We are thrilled to announce the Spring release of OpenRA!\n\nThis release, five and a half months in the making, is one of our biggest yet, with countless numbers of new features, bug fixes and improvements.  Sandworms and Carryalls for Dune 2000 and playable Countries in Red Alert are the most visible enhancements, but there is lots more to discover!\n\nCheck out http://openra.net for the full announcement and http://changelog.openra.net for the complete changelog.\n\nEnjoy and have a lot of fun! Thank you!
```

Comment:
- `with countless numbers of new features` -> `with countless new features`
 - you have 2 spaces before `Sandworms and Carryalls`